templates/zerver/for/working-groups-and-communities.md

```diff
@@ -0,0 +1,37 @@
+Working groups and communities need different things out of chat than
+companies do. At a company, users are on their company's chat all day. In a
+part-time working group or community, users might check the chat less than
```

Comment:
I think we might want to clarify that "many high-value participants might only have 30 minutes most weeks to participate in the organization".  I thin it's less about how often they check the chat than how to make the best use of their limited time.